experiment/test_nipopow.py

```diff
@@ -1,3 +1,17 @@
+# In order to run the test with sol 0.5.x, changes in the library file solidity.py are needed:
+# lines 57-60
+# ----------------------------------------
+# -    args = [
+# -        '--combined-json', combined,
+# -        '--add-std'
+# -    ]
+# ----------------------------------------
+# +    args = [
+# +        '--combined-json', combined #,
+# +        # '--add-std'
+# +    ]
+# ----------------------------------------
+
```

Comment:
I suggest let's not include this, make this PR about the smart contract only and let's get it merged today. Next step will be to use sth like a py-evm to rewrite this file and make sure tests work.

Reply:
Removed